diff -Naur a/plugins/daemon/irc.py b/plugins/daemon/irc.py
--- a/plugins/daemon/irc.py	2018-09-22 19:29:15.677064165 +0200
+++ b/plugins/daemon/irc.py	2018-09-22 19:30:18.980125473 +0200
@@ -50,6 +50,7 @@
                     'nickname': {'type': 'string'},
                     'channels': one_or_more(channel_pattern),
                     'nickserv_password': {'type': 'string'},
+                    'password': {'type': 'string'},
                     'invite_nickname': {'type': 'string'},
                     'invite_message': {'type': 'string'},
                     'task': one_or_more({
@@ -245,6 +246,7 @@
                          'invite_nickname': config.get('invite_nickname'),
                          'invite_message': config.get('invite_message'),
                          'nickserv_password': config.get('nickserv_password'),
+                         'password': config.get('password'),
                          'use_ssl': config.get('use_ssl')}
         SimpleIRCBot.__init__(self, ircbot_config)
 
